Extension/src/LanguageServer/Providers/workspaceSymbolProvider.ts

```diff
@@ -22,10 +26,21 @@ export class WorkspaceSymbolProvider implements vscode.WorkspaceSymbolProvider {
 
         // Convert to vscode.Command array
         symbols.forEach((symbol) => {
-            const suffix: string = util.getLocalizedString(symbol.suffix);
+            let suffix: string = util.getLocalizedString(symbol.suffix);
             let name: string = symbol.name;
             if (suffix.length) {
+                if (symbol.scope === SymbolScope.Private) {
+                    suffix = localize("c.cpp.symbolscope.separator", "{0}, {1}", "private", suffix);
+                } else if (symbol.scope === SymbolScope.Protected) {
+                    suffix = localize("c.cpp.symbolscope.separator", "{0}, {1}", "protected", suffix);
```

Comment:
Also, could you assign "private" or "protected" to a variable, so only one call to localize() is necessary?  Multiple calls with the same ID should be fine, but I'm not entire certain what happens in that case.

Reply:
I created a util function reused by all 4 cases.